src/contexts/Account.tsx

```diff
@@ -0,0 +1,106 @@
+import { Account, WalletMetadata } from '@polkadot-onboard/core';
+import type { Signer } from '@polkadot/api/types';
+import React, { useContext, createContext, useState, useEffect } from 'react';
+import { useWallets } from './Wallets';
+
+export type SigningAccount = {
+  account: Account;
+  signer: Signer;
+  sourceMetadata: WalletMetadata;
+};
+
+// account context
+const CONNECTED_ADRR_STORAGE_KEY = 'connectedAddress';
+const accountContext = createContext({});
+export const useAccount = () => useContext(accountContext);
+
+const AccountProvider = ({ children }: { children: React.ReactNode }) => {
+  const [_connectedAccount, _setConnectedAccount] = useState<SigningAccount>();
```

Comment:
Why the underscore?

Reply:
I needed to override these later before sending it in the context, so added underscore to use them as local states.
these are the ones that are set in the context:
https://github.com/jeluard/swipealot/blob/93c81eb1a85ac90109adcabfb3a07bcc27fcbc57/src/contexts/Account.tsx#L70
https://github.com/jeluard/swipealot/blob/93c81eb1a85ac90109adcabfb3a07bcc27fcbc57/src/contexts/Account.tsx#L89
 Should I rename?

Reply:
Mostly curious. Maybe worth a comment?